kclvm/parser/src/parser/stmt.rs

```diff
@@ -691,7 +692,8 @@ impl<'a> Parser<'_> {
                     decorators.push(node_ref!(x, expr_pos));
                 }
                 _ => {
-                    panic!("invalid Decorator: {:?}", expr);
+                    self.sess
+                        .struct_compile_error("Invalid Decorator", expr.get_pos());
```

Comment:
More friendly error messages are required. For example, which tokens should be expected and what a correct decorator looks like.